Annotations/Anonymize.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace SuperBrave\GdprBundle\Annotations;
+
+use Doctrine\Common\Annotations\Annotation;
+use Doctrine\Common\Annotations\AnnotationException;
+
+/**
+ * Annotation to flag an entity field to be anonymized
+ * to conform with the GDPR right to be forgotten
+ * 
+ * @Annotation()
+ * @Annotation\Target({"PROPERTY"})
+ */
+class Anonymize
+{
+    /**
+     * Alias of TYPE_CONCAT
+     */
+    const TYPE_STRING = 'string';
+
+    /**
+     * Anonymize a string to <columnName>-<primaryKey>
+     */
+    const TYPE_CONCAT = 'concat';
+
+    /**
+     * Anonymize a field by emptying it
+     * Alias of TYPE_FIXED with an empty value.
+     */
+    const TYPE_TRUNCATE = 'truncate';
+
+    /**
+     * Anonymize an IP field by filling in 127.0.0.1
+     * If this is a string field, INET_ATON will be used instead.
```

Comment:
What is ment by this comment? Is the mysql function of INET_ATON going to be used, why? I think this is not generic because not everybody is saving an ip-adres as a integer right?

Reply:
Yes and that is exactly my question because that is not generic. This type/annotation should not deal with how things are saved but should only know what to change.

Reply:
For now let's keep this as a string only field since that is the most common way to save IP addresses. We can then see how it will end up and decide on what we really want to support.

@stokjes brought forward a valid point where if we make an exception for INET_ATON there might be other ways of storing IP addresses and we might have to add more exceptions to support them!